src/login/pam_systemd.c

```diff
@@ -199,6 +199,18 @@ static int export_legacy_dbus_address(
         _cleanup_free_ char *s = NULL;
         int r = PAM_BUF_ERR;
 
+        /* FIXME: We *really* should move the access() check into the
+         * daemons that spawn dbus-daemon, instead of forcing
+         * DBUS_SESSION_BUS_ADDRESS= here. */
```

Comment:
i think we should conditionalize this as LEGACY or so? I don't think this is really worth supporting in new stuff

Reply:
I prefer not to make things more complicated than they need to be. The tribulations we went through with systemd-240 suggest that dropping this is going to have bad effects even on recently written software. I think it's OK to export the variable without any plans to remove it.